tangelo/toolboxes/operators/tests/test_taper_qubits.py

```diff
@@ -58,6 +58,19 @@ def test_z2taper_h2_jw_spinup_first(self):
 
         self.assertAlmostEqual(e, e_taper, places=5)
 
+    def test_z2taper_h4_jw_spinup_first(self):
+        """Test that a small qubit operator will be tapered properly when using tapering from a larger operator.
+        In this case, a 6 qubit operator from an 8 qubit operator tapering."""
+
+        qu_op = load_operator("H4_JW_spinupfirst.data", data_directory=pwd_this_test+"/data", plain_text=True)
+
+        tapering = QubitTapering(qu_op, 8, 4, 0, "JW", True)
+
+        try:
```

Comment:
What is the purpose of this test? Should we verify the result?

Reply:
It would seem James is happy as long as it does not raise an error, but I would be happier if the test checked for the output value or one of its properties (number of qubits or terms we expected to be present etc).